src/sql/src/plan.rs

```diff
@@ -939,6 +942,13 @@ pub struct ExplainTimestampPlan {
     pub raw_plan: HirRelationExpr,
 }
 
+#[derive(Debug)]
+pub struct ExplainSchemaPlan {
+    pub format: ExplainFormat,
```

Comment:
It seems this is unused.

Reply:
Ah right, copy pasta from other explains. I can remove this since only json is supported as of now.

Reply:
removed